packages/website/docs/docs/cursor-selections.md

```diff
@@ -17,23 +17,56 @@ The text cursor is the blinking vertical line you see when typing in the editor.
 ```typescript
 type TextCursorPosition = {
   block: Block;
+  prevBlock: Block | undefined;
```

Comment:
I'm curious; what do you see as main usecase for prevBlock / nextBlock? Why did we add this?

Reply:
Was necessary to add nextBlock to retain the old functionality for slash menu items. Since we changed how slash menu items are defined, I had to change the `execute` callback from using TipTap editor commands to BlockNote editor functions. One command that we used a lot was `BNUpdateOrCreateBlock`, in which the selection is set to the start of the next block. So I needed to add functions for changing the selection and getting the next block, and added prevBlock since it looked weird having only 1 of the 2 in the docs.